apertium-tur-uzb.tur-uzb.dix

```diff
@@ -3965,7 +3968,7 @@
        <e><p><l>kadar<s n="post"/></l><r>gacha<s n="post"/></r></p></e>
        <e><p><l>yoluyla<s n="post"/></l><r>orqali<s n="post"/></r></p></e>
        <e><p><l>yıldan<b/>yıla<s n="adv"/></l><r>yilgacha<s n="adv"/></r></p></e>
-       <e><p><l>kaç<s n="adv"/></l><r>necha<s n="adv"/></r></p></e>
+       <e><p><l>kaç<s n="det"/><s n="qnt"/></l><r>necha<s n="adv"/></r></p></e>
```

Comment:
What about the tags for necha?

Reply:
Fixed it (made `<s n="det"/><s n="qnt"/>`)